src/OnScreen/WallpapersSelector.vala

```diff
@@ -55,20 +55,22 @@ namespace Komorebi.OnScreen {
 			// create_new_thumbnail.clicked.connect(() => wallpaperChanged());
 
 			// addThumbnail(create_new_thumbnail);
-			// thumbnailsList.append(create_new_thumbnail); 
-			
+			// thumbnailsList.append(create_new_thumbnail);
+
 			add(grid);
 		}
 
 
 		public void getWallpapers () {
 
+			string package_datadir = Config.package_datadir;
+
 			clearGrid();
 
 			foreach(var thumbnail in thumbnailsList)
 				thumbnailsList.remove(thumbnail);
 
-			File wallpapersFolder = File.new_for_path("/System/Resources/Komorebi");
+			File wallpapersFolder = File.new_for_path(@"$package_datadir");
```

Comment:
Since this is just a single variable, you can just supply `package_datadir` directly.

Reply:
As it turns out, you can do exactly that. This is a great example of why I try not to code into the wee hours...

I've got a changed staged locally to fix this, because why not, but I'll wait to see if there's other feedback to bundle into the commit before I push.